runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -924,17 +911,15 @@ default. To start using more features you have to create a "vimrc" file.
 
  2. Type the start of a command: `:e`{vim}
 
- 3. Press `<C-d>`{normal} and Vim will show a list of commands that start
-     with "e".
+ 3. Press `<C-d>`{normal} and Vim will show a list of commands beginning with "e".
 
  4. Press `<Tab>`{normal} and Vim will complete the command name to ":edit".
 
  5. Now add a space and the start of an existing file name: `:edit FIL`{vim}
 
- 6. Press `<Tab>`{normal}. Vim will complete the name (if it is unique).
+ 6. Press `<Tab>`{normal}. Vim will complete the name (FIL -> FILE, if it is unique).
```

Comment:
`Vim will complete the name FIL to FILE if it is unique.`

You can make FIL,FILE standout with quotes or special characters to make them standout.